drivers/imx-enet.c

```diff
@@ -584,19 +653,23 @@ static uint16_t enet_mdioIO(enet_state_t *state, unsigned addr, unsigned reg, un
 
 #if !ENET_MDC_ALWAYS_ON
 	state->mmio->MSCR = state->mscr;
+	usleep(100); /* NOTE: needed for MDC to adjust to the change */
```

Comment:
MDIO doesn't work without it in MDC-not-always-on mode. Maybe we can consider checking for it with e.g. read frames?

Reply:
Okay, after some further testing, the MDC-not-always-on code path doesn't seem to be working reliably on neither imxrt1064 nor imx6ull, so a comment should be added or the functionality removed alltogether. Also I don't think this was tested, because it doesn't seem to be working even on [b1ab9f6](https://github.com/phoenix-rtos/phoenix-rtos-lwip/commit/b1ab9f6aa6fa37da945fe368c50334e2f8b71458).